kerml/src/examples/Simple Tests/Associations.kerml

```diff
@@ -0,0 +1,11 @@
+package Associations {	
+	assoc A {
+		end x;
+		end y[1..*];
+	}
+	
+	assoc B specializes A {
+		end x1;
+		end y1[0..*] redefines y;
```

Comment:
Is it correct and intended to widen the multiplicity `0..*` over `1..*` in the specialization?

Reply:
See the description in [ST6RI-519](https://openmbee.atlassian.net/browse/ST6RI-519). This issue was included on this branch because the resolution to [ST6RI-518](https://openmbee.atlassian.net/browse/ST6RI-518) (also included on the branch) was causing a spurious error message.

Reply:
OK. Do we explain the interpretation of multiplicities on association ends in the KerML specification? If not, I think we should raise a issue to do that, as this is always a tricky thing to understand for most people in my experience.

Reply:
The special nature of end multiplicities are defined in subclause 7.4.4.4 of the KerML spec. However, we have several issues already open on better specifying the semantics of association participant and end features. This is high priority semantic debt we will get back to eventually.